siege_utilities/geo/gazetteers/nominatim_gazetteer.py

```
@@ -0,0 +1,373 @@
+            country=country.upper() if country else None,
+            admin_levels=_admin_levels(addr),
+            source=self.provider_name,
+            raw=row,
+        )
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def _geometry_from_row(row: dict[str, Any]) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_geometry_from_row`

(ANN401)